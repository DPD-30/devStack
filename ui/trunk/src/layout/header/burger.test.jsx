import React from 'react'
import { renderToString } from 'react-dom/server'


import LayoutBurger from './burger.jsx'

jest.unmock('./burger.jsx')


test('LayoutBurger matches snapshot', () => {
	const comp = renderToString(<LayoutBurger />)

	expect(comp).toMatchSnapshot()
})
