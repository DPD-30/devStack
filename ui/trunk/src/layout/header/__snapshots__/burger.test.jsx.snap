// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutBurger matches snapshot 1`] = `
<button class="headerBurger btn btn-secondary"
        data-reactroot
>
  <span aria-hidden="true"
        class="fa fa-bars"
  >
  </span>
</button>
`;
